build-system/common/dom-svg-attributes.js

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
This file lacks context. 

- Name it `preact-prop-names.js`.
- Add a `@fileoverview` comment explaining that it contains mappings between React and Preact style props, including this URL: `https://preactjs.com/guide/v10/differences-to-react/#raw-html-attributeproperty-names`
- Rename the constants based on the style reflected by the keys: `DOM_ATTRIBUTES` should be `DOM_ATTRIBUTES_REACT` and `DOM_ATTRIBUTES_FLIPPED` should be `DOM_ATTRIBUTES_PREACT`.